src/pages/InstitutionSettings/InstitutionSettings.tsx

```diff
@@ -38,8 +43,13 @@ const InstitutionSettings: FC = () => {
           workingTimes={
             <WorkingTimes name={name} id={institutionId} data={institution} />
           }
-          //TODO: add vacation days component as prop
-          //vacationDays={<div>Puhkepäevad:</div>}
+          vacationDays={
```

Comment:
**[non-blocker]**

Not really related to this task, but wondering why workingTimes and vacationDays are added as props to InstitutionForm, instead of just importing them there. InstitutionForm is not used anywhere else and never has any other components for these props

Reply:
Yep interesting, maybe there were some different plans earlier, regardless we are importing them from InstitutionForm now